client.go

```diff
@@ -266,10 +267,21 @@ func (c *Client) Go(method string, args interface{}, reply interface{}, done cha
 	return call
 }
 
+// Call invokes the named function, waits for it to complete, and returns its error status.
+func (c *Client) CallWithContext(method string, args interface{}, reply interface{}, ctx context.Context) error {
```

Comment:
For functions that takes a Context argument, usually the convention is to pass that as first one. Can you refactor and make that change please?

Example: https://golang.org/pkg/net/http/#NewRequestWithContext

Reply:
@cenkalti done, and fixed the docblock too.